### ISSUE TITLE
Hydroponics Unit should produce more oxygen

### ISSUE THREAD

**Issue by Fanasuul** [GTNewHorizons/GT-New-Horizons-Modpack#17549]
### Your GTNH Discord Username

Fanasuul#6878

### Your Pack Version

2.6.0

### Your Proposal

Currently the Hydroponics Unit when vanilla Seeds are used (tried some other seeds and none worked) produces only 6 oxygen/s. Considering the production cost of one such unit it seems this value should be significantly higher.

### Your Goal

Make Hydroponics Unit worth it

### Your Vision

It should at least be able to support a Bubble Distributor and Oxygen Compressor = 200 o/s

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by dagger8243** [comment]
some1 else actually using these things ?

**Comment by Fanasuul** [comment]
Yes, I figured I might be the only one but they should at least be usable
